pl_examples/utils/loaders.py

```diff
@@ -0,0 +1,77 @@
+from threading import Thread
+from queue import Queue
+
+import torch
+from torch import Tensor
+import torch.nn as nn
+import torch.nn.functional as F
+from torch.utils.data import DataLoader
+
+
+class AsynchronousLoader(DataLoader):
+    """
+    Class for asynchronously loading from CPU memory to device memory
+
+    Parameters
+    ----------
+    dataset: PyTorch Dataset
+        The PyTorch dataset we're loading
+    device: PyTorch Device
+        The PyTorch device we are loading to
+    num_workers: Integer
+        Number of worker processes to use for loading from storage and collating the batches in CPU memory
+    queue_size: Integer
+        Size of the queue used to store the data loaded to the device
+    """
+
+    def __init__(self, dataset, device, num_workers=8, queue_size=10, **kwargs):
+        super(AsynchronousLoader, self).__init__(dataset=dataset,
+                                                 pin_memory=True,
+                                                 num_workers=num_workers,
+                                                 **kwargs
+                                                 )
```

Comment:
```suggestion
        super().__init__(dataset=dataset, pin_memory=True, num_workers=num_workers, **kwargs)
```
just to be consistent...

Reply:
Already dealt with this